store/sqlstore/team_store.go

```diff
@@ -643,6 +643,7 @@ func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, restricti
 	query := s.getTeamMembersWithSchemeSelectQuery().
 		Where(sq.Eq{"TeamMembers.TeamId": teamId}).
 		Where(sq.Eq{"TeamMembers.DeleteAt": 0}).
+		OrderBy("UserId").
```

Comment:
This seems like a breaking change, since the natural order might have closely approximated `CreateAt` previously, but now will be effectively random.

Can we pull this out into a separate PR for review to avoid this being buried?

Reply:
The previous existing version was unordered, and was checking as ordered by the app layer tests, which hasn't the same behavior for postgres.

The expected order is extracted from the tests, I modified it, and the tests now is working because the order expected is by userId. I guess mysql was using the TeamId+UserId key to fetch the data so the order used was that, but postgres must be following another appraoch and is not getting the same order. I'm ok moving this to another PR.

Reply:
The order here is not specially important meanwhile we can assure an order, because if not, the pagination itself makes no sense.

Reply:
MySQL already orders by UserId because of primary key so this change affects only Postgres and makes the result same as with MySQL.

Reply:
I'm almost sure that is not true for any SQL database out there, MySQL included. The absense of order doesn't grantee you any order at all, the DBMS is going to use the most efficient order he can, and that depends on the query, the joins, the indexes used, and a lot of other factors. The standard approach for any SQL database is to use `ORDER BY` if you need things ordered  somehow.

Reply:
I meant that this orderby doesn't change the results for mysql so is not a breaking change. And it should be added.